configurations/manager/debian10.yaml

```diff
@@ -1,2 +1,4 @@
 ami_id_monitor: 'ami-07d02ee1eeb0c996c'  # Debian buster image - debian-10-amd64-20210208-542
 ami_monitor_user: 'admin'
+
+manager_scylla_backend_version: '2021'  # Notice: Uses 2021.1, while other (newer deb) use 2022, since we support both
```

Comment:
understand what is written in the comment, but it should be in a different commit? or to change the commit message to be something like "bump Scylla backend version for manager jobs"